sklearn/model_selection/__init__.py

```diff
@@ -1,32 +1,34 @@
+from ._search import GridSearchCV
+from ._search import UnsupervisedGridSearch
+from ._search import ParameterGrid
+from ._search import ParameterSampler
+from ._search import RandomizedSearchCV
+from ._search import UnsupervisedRandomizedSearch
+from ._search import fit_grid_point
```

Comment:
Actually ignore this comment. This looks good and this is just in the `__init__.py`. Apologies for the noise!!